ts/session/messages/outgoing/content/sync/SentSyncMessage.ts

```diff
@@ -0,0 +1,67 @@
+import { SyncMessage } from './SyncMessage';
+import { SignalService } from '../../../../../protobuf';
+import { MessageParams } from '../../Message';
+import { PubKey } from '../../../../types';
+
+interface SentSyncMessageParams extends MessageParams {
+  dataMessage: SignalService.DataMessage;
+  expirationStartTimestamp?: number;
+  sentTo?: [PubKey];
+  unidentifiedDeliveries?: [PubKey];
```

Comment:
```suggestion
  unidentifiedDeliveries?: Array<PubKey>;
```

need to make change in the variable inside SentSyncMessage too

Reply:
fixed